app/main.py

```diff
@@ -1 +1,50 @@
-#  write your code here
+from __future__ import annotations
+
+
+class Animal:
+
+    animals = []
+
+    def __init__(self, name: str,
+                 appetite: int,
+                 is_hungry: bool = True) -> None:
+        self.name = name
+        self.appetite = appetite
+        self.is_hungry = is_hungry
+        Animal.animals.append(self)
+
+    def print_name(self) -> None:
+        print(f"Hello, I'm {self.name}")
+
+    def feed(self) -> int:
+        if self.is_hungry:
+            print(f"Eating {self.appetite} food points...")
+            self.is_hungry = False
+            return self.appetite
+        return 0
```

Comment:
its ok but in cases like this it will be more understandable to write reversed
`if not self.is_hungry`

Reply:
Thank you, changed it